src/net/udp.rs

```diff
@@ -511,6 +511,13 @@ impl UdpSocket {
         self.inner.leave_multicast_v6(multiaddr, interface)
     }
 
+    /// Get the value of the `IPV6_V6ONLY` option on this socket.
+    #[cfg(all(unix, feature = "os-poll"))]
```

Comment:
Windows has this as well, see https://github.com/rust-lang/socket2/blob/35c26913ec3c0205fd1d9c3946ec0abcb50e3a4c/src/socket.rs#L1170-L1175.

And I'm afraid this needs a fake implementation in the shell, e.g. see: https://github.com/tokio-rs/mio/blob/ec28a8dcdd09bf2d7ad4030b7330c86834827124/src/sys/shell/tcp.rs#L16-L18